js/components/utils.js

```diff
@@ -882,6 +882,48 @@ var Utils = {
 
     return coords;
   },
+
+  /**
+   * get mobile device orientation type
+   * @returns {string} - one of the following:
+   * "portrait-primary"
+   * "portrait-secondary" (portrait upside down)
+   * "landscape-primary"
+   * "landscape-secondary" (landscape upside down)
+   */
+  getOrientationType: function() {
+     var orientationType = window.screen.orientation;
+     if (orientationType && orientationType !== null && typeof orientationType === 'object') {
+       orientationType = orientationType.type;
+     } else {
+       orientationType = window.screen.mozOrientation || window.screen.msOrientation;
+     }
+     return orientationType;
+  },
+
+  /**
+   * if device orientation is portrait set "landscape-primary" orientation for the device
+   * @returns {boolean} - true if "landscape-primary" orientation was set, returns false in otherwise
+   */
+  setLandscapeScreenOrientation: function() {
+    var orientationType = this.getOrientationType();
+    if (orientationType && (orientationType === "portrait-secondary" || orientationType === "portrait-primary")) {
```

Comment:
Should we create an "isPortrait" helper function, checking orientationType === "portrait-secondary" || orientationType === "portrait-primary"?

Reply:
This is not necessary because it is not used in any other places.